esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2214,8 +2229,37 @@ bool CFileSprayEx::onReplicate(IEspContext &context, IEspReplicate &req, IEspRep
     return true;
 }
 
+void CFileSprayEx::getDropZoneInfoByDestPlane(double clientVersion, const char* destGroup, const char* destFileIn, StringBuffer& destFileOut, StringBuffer& umask)
```

Comment:
is destGroup the plane? if so param should be renamed

Reply:
The destGroup is the dropzone name - in the case of the containerized code, it's the plane. I have not renamed the field as it is used in lots of places as destGroup and I don't want to cause confusion.